### DISCUSSION TITLE
Let the user add the search bar of any website to Flow Launcher

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1173] [Ideas]
There are plenty of websites that provide specialized search results. For example, Wikipedia, wiktionary, Amazon, Twitter, etc.

Let the user click inside the search bar of any website and add it directly to Flow Launcher.
In the next step, he should be able to select his keyboard shortcut for this website search.

After that, he should be able to launch that website in the default browser and pass the search string to the search bar of that website.

This functionality is provided in a Chrome addon called [SearchBar](https://chrome.google.com/webstore/detail/searchbar/fjefgkhmchopegjeicnblodnidbammed).

**Comment by raindropsfromsky** [comment]
I found that this functionality is provided in the **web searches** plugin. However, it requires us to compose the search, copy the URL from the browser's address bar and then paste it in the **Web Searces** plugin. Then the search string has to be replaced with {q}.

This looks like a workaround, and not as elegant as the **SearchBar** addon of Chrome.

**Reply by taooceros** [reply]
I think the SearchBar addon simply embed some of the search source, which we have also done. The url format is required to provide the maximized customization since we have no way to know the api design for a specific search source.
